### DISCUSSION TITLE
Can't open preferences - Taskbar icon missing

### DISCUSSION THREAD

**Discussion by faid0n** [Ulauncher/Ulauncher#921] [General]
Wanted to change something in the preferences, but there is now way to access them since the taskbar icon (applet) is not shown.
It seems to me that having that applet there is the only way of opening preferences (which if true is not very smart imo. Please correct me if that is not the case).
Something like #729 would have basically solved my problem by just not needing that applet in the first place.

Anyway so I have reinstalled it (newest version) and the icon still is not present. Any ideas?

Linux Mint 20.1

**Comment by friday** [comment]
The taskbar icon isn't needed at all. You should have a cogwheel icon that takes you to the settings as seen below. If you don't, there's something else that's wrong here. Have you tried clicking in this area in case it's just a visual bug but there's an actual button there?

![2021-06-14_01-27](https://user-images.githubusercontent.com/515120/121824899-c74d3100-ccaf-11eb-9815-1911032408ce.png)
